staging/src/k8s.io/sample-apiserver/pkg/cmd/server/start.go

```diff
@@ -141,11 +141,9 @@ func (o *WardleServerOptions) Config() (*apiserver.Config, error) {
 	serverConfig.OpenAPIConfig.Info.Title = "Wardle"
 	serverConfig.OpenAPIConfig.Info.Version = "0.1"
 
-	if utilfeature.DefaultFeatureGate.Enabled(features.OpenAPIV3) {
```

Comment:
We are no longer using `features` so we need to remove the corresponding import `k8s.io/apiserver/pkg/features` in this file above.

Reply:
Corrected.